options/locale/locale_en-US.ini

```diff
@@ -677,8 +679,11 @@ comment_type_group_project = Project
 comment_type_group_issue_ref = Issue reference
 saved_successfully = Your settings were saved successfully.
 privacy = Privacy
-keep_activity_private = Hide Activity from profile page
-keep_activity_private_popup = Makes the activity visible only for you and the admins
+actions_visibility_popup = Specify which activities are visible
+actions_visibility = Visible activities
+actions_visibility.public_popup = Only activies from repositories which can be accessed by the viewer are visible
+actions_visibility.all_popup = All actions are visible, also from repositories can't be accessed by the viewer
```

Comment:
Maybe we need some explanations about whether repository or organization name is visible.